.github/workflows/build.yml

```diff
@@ -7,8 +7,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [10.x, 12.x]
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        node-version: [12.x]
+        os: [ubuntu-latest, windows-latest]
```

Comment:
```suggestion
        os: [ubuntu-latest]
```

Reply:
The windows one is important I think because we support execution on windows, and there is code in the SDK dealing with path differences between the platforms.

Reply:
Thanks for the explanation!